docs/api.md

```diff
@@ -31,8 +31,11 @@ curl --header "Authorization: Bearer ${KUBERNETES_USER_TOKEN}" \
      "https://${K8S_API_URL}/apis/token.kubevirt.io/v1alpha1/namespaces/${VMI_NAMESPACE}/virtualmachines/${VMI_NAME}/vnc?duration=${DURATION}"
 ```
 
-The `KUBERNETES_USER_TOKEN` variable is a bearer token used to authenticate with
-kubernetes API. It can be obtained using:
+In this example, we use a bearer token to authenticate the user with the Kubernetes API server, because it was simple.
```

Comment:
I would not add `because it was simple.` sentence

Reply:
done